docs/troubleshooting.md

```diff
@@ -6,7 +6,7 @@ The Infrastructure agent installs bundled binaries together in the following dir
 
 ```shell
 # Linux
-/var/db/newrelic-infra/newrelic-integrations/bin/
+/opt/newrelic-infra/newrelic-integrations/bin/
```

Comment:
Maybe it would be good to add both paths and versions (>=1.48.0)

Reply:
hmm yes, fixing!

Reply:
Done in https://github.com/newrelic/nri-flex/pull/454/commits/d16bdf445f61bb29ba46dae6a4ed87a741ced539!